src/main/java/banking/database/AccountDao.java

```diff
@@ -0,0 +1,97 @@
+package banking.database;
+
+import banking.model.Account;
+import banking.model.Card;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class AccountDao {
+
+    public static final String SQL_GET = "SELECT * FROM account WHERE cardNumber = ?;";
+    public static final String SQL_GET_IF_CORRECT = "SELECT * FROM account WHERE cardNumber = ? AND cardPin = ?;";
+    public static final String SQL_SAVE = "INSERT INTO account(cardNumber, cardPin) VALUES(?, ?);";
+    public static final String SQL_UPDATE_BALANCE = "UPDATE account SET balance = balance + ? WHERE cardNumber = ?";
+    public static final String SQL_DELETE = "DELETE FROM account WHERE id = ?;";
+
+    private final DBManager dbManager;
+
+    public AccountDao(DBManager dbManager) {
+        this.dbManager = dbManager;
+    }
+
+    public Optional<Account> get(String cardNumber) {
+        try (PreparedStatement statement =
+                     dbManager.getConnection().prepareStatement(SQL_GET)) {
+            statement.setString(1, cardNumber);
+
+            ResultSet rs = statement.executeQuery();
+            if (rs.next()) {
+                return Optional.of(new Account(
+                        rs.getInt("id"),
+                        new Card(rs.getString("cardNumber"), rs.getString("cardPIN")),
+                        rs.getInt("balance")
+                ));
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
?

Reply:
Done. Now RuntimeException is thrown with message and caused exception that forces program to stop and show message